src/App/App.js

```diff
@@ -606,6 +580,14 @@ function FullApp() {
             />
             <div className="App-slippage-tolerance-input-percent">%</div>
           </div>
+          {parseFloat(slippageAmount) < (DEFAULT_ALLOWED_SLIPPAGE_BPS / BASIS_POINTS_DIVISOR) * 100 && (
+            <div className="warning settings-modal-error">
+              <Trans>
+                Allowed Slippage below {(DEFAULT_ALLOWED_SLIPPAGE_BPS / BASIS_POINTS_DIVISOR) * 100}% may result in
```

Comment:
I know `formatPercentage` works with BigNumbers but wouldn't it still better to use it? To keep consistency in formatting

Reply:
Also I'm not sure I fully get the idea of BASIS_POINTS_DIVISOR here.
Why not just make DEFAULT_ALLOWED_SLIPPAGE = 0.3?

Reply:
Using `formatPercentage` here would make the code unnecessarily complicated and adds overhead, especially given the limitations with decimals and `BigNumber`.

Regarding the use of `BASIS_POINTS_DIVISOR`, it ensures that our internal calculations remain consistent throughout the app. We store `DEFAULT_ALLOWED_SLIPPAGE` in BPS, which stands for "basis points." This is a commonly used term in finance to signify changes in value. Also helps to store integers in local storage.

Reply:
ok, makes sense